src/App.tsx

```diff
@@ -1,26 +1,43 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+/* eslint-disable max-len */
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/store';
+import { todosSlice } from './features/todos';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  const [isLoaderActive, setIsLoaderActive] = useState(false);
+
+  useEffect(() => {
+    setIsLoaderActive(true);
+    getTodos()
+      .then(res => dispatch(todosSlice.actions.setTodos(res)))
+      .finally(() => setIsLoaderActive(false));
+  }, []);
+
+  return (
+    <>
+      <div className="section">
+        <div className="container">
+          <div className="box">
+            <h1 className="title">Todos:</h1>
 
-          <div className="block">
-            <Loader />
-            <TodoList />
+            <div className="block">
+              <TodoFilter />
+            </div>
+
+            <div className="block">
+              {isLoaderActive ? <Loader /> : <TodoList />}
```

Comment:
Ensure that the `Loader`, `TodoFilter`, and `TodoList` components are correctly implemented and imported. They should handle the loading state and display the todos list as expected.